lib/fpm/package/tar.rb

```diff
@@ -49,7 +49,7 @@ def input(input_path)
   def output(output_path)
     output_check(output_path)
     # Unpack the tarball to the staging path
-    args = ["-cf", output_path, "-C", staging_path]
+    args = ["--owner=root", "--group=root", "-cf", output_path, "-C", staging_path]
```

Comment:
Yep. Like I said, I included this line in this pull request in error, It doesn't belong anywhere other than my fork.